lua/treesitter-context.lua

```diff
@@ -799,6 +802,23 @@ function M.setup(options)
   end
 end
 
+function M.get_contexts()
+  return contexts
+end
+
+function M.previous_context()
```

Comment:
I'm not really comfortable exposing the internals of the context object. Can we add functions for simple navigation instead?

Reply:
sure! how do you see the API? something like `M.go_next_context` and 'M.go_to_previous_context`?

Reply:
`goto_next` doesn't make much sense since that information isn't available. I think just `go_to_previous_context()` (no arguments).

Reply:
updated and rebased.